### PR TITLE
Feat-live 10403 btc doc

### PR DESCRIPTION
**Pull request by hzheng-ledger**
### 📝 Description

High level design documentation for bitcoin family

### ❓ Context

- **JIRA or GitHub link**: 

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [X] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** No tests need as only documentation is added in the PR
- [ ] **Impact of the changes:** No impact
  - ...

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/families/bitcoin/docs/BitcoinSync.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+1. **Synchronize All Receiving Addresses:** Derived from `m / purpose' / coin_type' / account' / 0`.
+2. **Synchronize All Change Addresses:** Derived from `m / purpose' / coin_type' / account' / 1`.
+3. **Batch Synchronization:** In both step 1 and 2, addresses are synchronized in batches of 20. For example, receiving addresses from `m / purpose' / coin_type' / account' / 0 / 0` to `m / purpose' / coin_type' / account' / 0 / 19` are synchronized first. If any address in these 20 has a transaction, the next batch of 20 addresses is synchronized, continuing until a batch with no transactions is found.
+4. **Synchronization Details:**
+    - Addresses are obtained using `this.crypto.getAddress(this.derivationMode, this.xpub, account, index)`.
+    - All confirmed transactions for an address are fetched from `https://explorers.api.live.ledger.com/blockchain/v4/[coin_ticker]/address/[address]/txs`.
+    - All pending transactions are fetched from `https://explorers.api.live.ledger.com/blockchain/v4/[coin_ticker]/address/[address]/txs/pending`.
+    - All transactions for an address are summarized and UTXOs are calculated and stored in `storage`.
```
Comment (hedi-edelbloute) [inline libs/ledger-live-common/src/families/bitcoin/docs/BitcoinSync.md:18]: Maybe just a bit of information on how we estimate balance based on utxos data ?

**Review by hedi-edelbloute — APPROVED** [review]
Excellent ! thank you :+1:

File: libs/ledger-live-common/src/families/bitcoin/wallet-btc/pickingstrategies/Merge.ts [hunk 2]
```diff
@@ -8,6 +8,9 @@ import * as utils from "../utils";
 import { log } from "@ledgerhq/logs";
 import { OutputInfo } from "..";
 
+/**
+ * prioritizes the smallest UTXOs
```
Comment (lvndry) [inline libs/ledger-live-common/src/families/bitcoin/wallet-btc/pickingstrategies/Merge.ts:12]: What means "smallest" here ?

Comment (hzheng-ledger) [reply libs/ledger-live-common/src/families/bitcoin/wallet-btc/pickingstrategies/Merge.ts:12]: it means "prioritize using UTXOs with smaller denominations"
I've changed the wording now to be more accurate.

**Pull request merged**